tests/unit/dojson/test_utils.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
```

Comment:
File should be renamed `tests/unit/dojson/test_dojson_utils.py`, otherwise it will conflict with any other file called `test_utils.py`. A bit annoying.